### DISCUSSION TITLE
newbie: how to set ssh as default connection instead of windows cmd?

### DISCUSSION THREAD

**Discussion by walteroa2023** [Eugeny/tabby#8986] [Q&A]
Hi all,
sorry for my questions, but im newbie with tabby.
each time I open tabby, the window goes to cmd terminal.
How can I change this behavior, in order to select my configured profiles?

![image](https://github.com/Eugeny/tabby/assets/142554546/6f0901ff-94fb-4281-82b5-e508454d81d8)

Thx in advance!

**Comment by Eugeny** [comment]
You can change the default profile in Settings - Ptofiles

**Comment by walteroa2023** [comment]
I did it. I've configured ssh as default profile, but if "open a terminal at app startup" is enable, it comes again to CMD terminal.

I wanna just prompt for a new ssh connection, not switch manually to ssh from profile list...
